components/rmrk/Create/CreateCollection.vue

```diff
@@ -38,15 +38,24 @@
       </template>
 
       <template #footer>
-        <NeoField
-          v-if="isLogIn"
-          variant="danger"
-          :message="balanceNotEnoughMessage">
+        <NeoField v-if="isLogIn">
           <SubmitButton
             expanded
-            label="create collection"
+            :label="balanceNotEnough ? 'not enough funds' : 'create collection'"
             :loading="isLoading"
+            :icon="'false'"
```

Comment:
Does the icon actually exist?
```suggestion
            icon="'false"
```

Reply:
The prop takes a string and the default icon is `paper-plane`. My solution to disable the icon was to pass a non-existent one.

Reply:
ok, then we could set `icon=""` or add a `disabledIcon` props for it.

Reply:
I think the second option is the better solution.